### DISCUSSION TITLE
Are the words server side and client side misplaced in /docs/4.1/components/forms?

### DISCUSSION THREAD

**Discussion by sagarvd01** [twbs/bootstrap#30688] [General] [answered]
In the page [/docs/4.1/components/forms/](https://getbootstrap.com/docs/4.1/components/forms/), there is a section named **Server side** and it's content is as follows

>We recommend using client side validation, but in case you require server side, you can indicate invalid and valid form fields with .is-invalid and .is-valid. Note that .invalid-feedback is also supported with these classes.

I believe the words server side and client side is misplaced here because the recommended method is server side validation (because client side validation can easily bypassed) and as mentioned in the paragraph, we can't access the class names from server side.

**Comment by mdo** [comment, marked as answer]
Nope the words here are intentional and correct. There was a PR about this, and this would be more appropriate as an issue and not a discussion :).
